Allow for Scanner Gates to be used with Voice Analyzers

Axanthic opened the pull request:
Changes to be committed:
	modified:   code/game/machinery/scan_gate.dm

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Gates will now say 'BUZZZ!' allowing for them to be used with voice analyzers!
I've tested and it works as intended.
I'm unsure if 'exclaims', is the correct thing to use here please let me know!
![image](https://user-images.githubusercontent.com/19522166/88283983-9096b280-ccba-11ea-9733-90b4439112cf.png)
![bGLooTyMVR](https://user-images.githubusercontent.com/19522166/88283736-05b5b800-ccba-11ea-8205-f712845b66d0.gif)

## Why It's Good For The Game
Adds a bunch of possibilities when using scan gates, perhaps automatically locking doors or detonating bombs!

## Changelog
:cl:
Added the ability to use voice analyzers with scanner gates!
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

SheldonCurtiss committed:
On branch scannergates
 Changes to be committed:
	modified:   code/game/machinery/scan_gate.dm

XDTM commented:
If we eventually add reverse wires to machinery, that send signals through signallers when something happens, we could enable a lot of interesting interactions like this one. You could do stuff like tripping alarms when a door is opened or unbolted.

RaveRadbury commented:
Are the scanners making a buzz on a successful scan? Shouldn't there be a "success" and "failure" scan noise?

SheldonCurtiss committed:
Changes to be committed:
	modified:   code/__DEFINES/wires.dm
	modified:   code/datums/wires/_wires.dm
	modified:   code/game/machinery/scan_gate.dm
	modified:   tgstation.dme
